<?xml version="1.0"?>
<launch>
  <!-- start Webots -->
  <arg name="no-gui" default="false," doc="Start Webots with minimal GUI"/>
  <include file="$(find webots_robot)/launch/webots.launch">
    <arg name="mode" value="realtime"/>
    <arg name="no-gui" value="$(arg no-gui)"/>
    <arg name="world" value="$(find webots_robot)/worlds/pioneer3at.wbt"/>
  </include>

  <include file="$(find webots_robot)/launch/description/pioneer3at_description.launch"/>

  <!--arg name="auto-close" default="false" doc="Startup mode"/-->
  <node name="webots_robot" pkg="webots_robot" type="webots_robot_test" output="screen"/>
</launch>
